Submit branch

sinansevgi opened the pull request:
# Description
Youtube video player page clone created with using HTML semantic. 
reset.css file added to the project to reset default browser styling.

# How Has This Been Tested?

We used htmlhint and stylelint linters to check code for errors.

zdrale committed:
Organized css

sinansevgi committed:
Css File Organized

Css file organized by sections

sinansevgi committed:
Linter Files

Linter configuration files created

sinansevgi committed:
Create linters

sinansevgi committed:
Create .hintrc

sinansevgi committed:
Create .stylelintrc.json

sinansevgi committed:
Readme file created

sinansevgi committed:
Delete unnecessary action settings

Delete unnecessary action settings

sinansevgi committed:
Added missing file extension

Added missing file extension

sinansevgi committed:
Screenshot added

sinansevgi committed:
Case consistency fixed

sinansevgi committed:
gitignore file created

sinansevgi committed:
IDE dot folders removed

sinansevgi committed:
Merge branch 'master' into Submit-branch

arslanbisharat commented on style.css:
```diff
@@ -232,22 +195,45 @@ a > u {
 }
 
-.commentSection {
+/* Recommendation Section */
```
- [ ] remove commit that is not necessary

arslanbisharat commented on style.css:
```diff
@@ -140,68 +129,43 @@ nav {
+  padding: 10px;
+}
+
+/* Comment Section */
```
- [ ] remove commit that is not necessary

arslanbisharat commented on style.css:
```diff
@@ -98,13 +80,27 @@ nav {
 }
 
-/* video */
+/* Video section */
```
- [ ] remove commit that is not necessary

arslanbisharat commented on style.css:
```diff
@@ -98,13 +80,27 @@ nav {
+  width: auto;
+}
+
+/** Navigation Section **/
```
- [ ] remove commit that is not necessary

arslanbisharat commented on style.css:
```diff
@@ -72,7 +45,16 @@ nav {
+  margin-left: 10px;
+}
+
+/** Search Section **/
```
- [ ] remove commit that is not necessary

arslanbisharat commented on style.css:
```diff
@@ -1,3 +1,4 @@
+/* Global Section */
```
- [ ] remove commit that is not necessary

arslanbisharat reviewed (changes requested):
### Request changes

I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
It's okay. We are here to learn. Don't get discouraged. Work more hard.
**Please have a look at the suggestions to get more understanding. 👀**
- [x] Make PR more descriptive. 
- [x] remove a comment that is not necessary

- [x] [optional] Either remove GitHub actions from this repo because it is not necessary for this project or correct linter errors.

- [x] Your readme is not that descriptive please work on it.

- [x]  Increase the size of the search icon
![icon-size](https://user-images.githubusercontent.com/58504655/93318201-d9389b00-f827-11ea-9f6e-b6b102f23e87.jpg)

- [x] increase space and make it somehow similar to the original one.
![space](https://user-images.githubusercontent.com/58504655/93318332-02592b80-f828-11ea-97fd-f0b066e2ec0a.jpg)

- [x] Decrsese margin on the left side of the page to make it more attractive.

- [x] Align the items correctly.
![align](https://user-images.githubusercontent.com/58504655/93318567-48ae8a80-f828-11ea-967d-237ee510e01c.jpg)

- [x] Please manage to align text correctly in the box as given below  
![font](https://user-images.githubusercontent.com/58504655/93318680-6ed42a80-f828-11ea-96cd-f884f53b0779.jpg)

- [x] Add description to the repo
![description](https://user-images.githubusercontent.com/58504655/93318766-83182780-f828-11ea-8d58-0d972b32a46d.jpg)

- [x] Make comment section correct as given below or use orignal page
![comments](https://user-images.githubusercontent.com/58504655/93318847-9a571500-f828-11ea-8c75-6b1502c002c8.jpg)


**Please make the changes and request re-review for the given project**
Stay safe😇
-------------------------------------------------------------------------------------------------------------------
Happy Coding
Regards,
Arslan Bisharat
slack: arslanbisharat

sinansevgi committed:
Styling fixes 

Search button size fixed
Submenu items aligned to center
'View More' anchor added after comments
Unnecesarry comments removed.

sinansevgi committed:
Style changes made

increased space between video and recommendations section

sinansevgi committed:
Linter removed from project

zdrale committed:
Rearanged the order of the CSS files

arslanbisharat reviewed (approved):
### Congratulations APPROVED :clap:
Hello Team! I can see you worked really hard on this project. I reviewed your and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
Now you can merge the branches and move forward with your curriculum but before that 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 
**Happy Coding**:smiley:
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) @slack: arslanbisharat
